app/controllers/api/v1/todos_controller.rb

```diff
@@ -1,6 +1,17 @@
 class Api::V1::TodosController < Api::ApiController
   rescue_from ActiveRecord::RecordNotFound, with: :not_found_response
 
+  def create
+    user = User.find_by_auth_token(request.headers["Authorization"])
+
+    if user
+      user.todos.create(description: params[:description])
+      head :no_content
```

Comment:
"201 Created" would be a closer fit to represent what happens on a successful request here. 

When responding to successful create API requests, consider including a response body of the created todo (as JSON). This response will include the new todo's id which the client might find useful, for updating the view to include edit/delete functionality for the new resource.